packages/react-composites/src/composites/CallWithChatComposite/adapter/TestUtils.ts

```diff
@@ -276,7 +276,8 @@ export class MockCallAgent implements CallAgent {
   connectionState = 'Disconnected' as ConnectionState;
   kind = 'CallAgent' as CallAgentKind;
   emitter = new EventEmitter();
-  feature;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  feature: any;
```

Comment:
CallAgent doesn't have `feature`, should be able to remove this

Reply:
CallAgent extends CallAgentCommon which has feature
![image](https://github.com/Azure/communication-ui-library/assets/97130533/cf190cda-6619-41ec-9175-927165ac27cd)

Reply:
Ah so it is! Could we do something like this then?
```ts
feature: CallAgentCommon['feature'] = () => {
  throw Error('Method not implemented.');
};
```

i.e. use `CallAgentCommon['feature'] ` as the type

Reply:
unfortunately CallAgentCommon is private so no can do there. 
But we can do:
```typescript
feature: CallAgent['feature'] = () => {
  throw Error('Method not implemented.');
};
```

Reply:
_also, great suggestion!!_